utils/array.js

```diff
@@ -18,6 +18,21 @@ function chunks(array, size = 1) {
   return result;
 }
 
+/**
+ * Checks if two arrays have any common items
+ * @param array1 {Array} - first array
+ * @param array2 {Array} - second array
+ * @returns {boolean} - true if the arrays have at least one common item, false otherwise
+ */
+
+function arraysHaveCommonItem(array1, array2) {
+  if (!Array.isArray(array1) || !Array.isArray(array2) || !array1.length || !array2.length) {
```

Comment:
what are we trying to check here?

Reply:
If one of the array don't have any length or is not an array then I am returning false instead of looping and finding common items.

Reply:
do we need to check Array.isArray ?

can we just check if the array1?.length property exists on the array if yes then it will return false since 0 is falsy in case array is empty & if its not it will continue to execute your written logic

Reply:
sure will do it.